docker/instant-seal-node.overview.md

```diff
@@ -1,5 +1,12 @@
 # Frequency Collator Node in Local Only Sealing Mode
 
+## Deprecation Note
+
+This image has been deprecated and will be removed in a future Frequency release. The new image, [frequencychain/standalone-node](./standalone-node.overview.md), replaces it. Current usage patterns for this image are fully supported by the new image, as well as additional enhancements.
```

Comment:
I think this should be communicated to other teams such as **custodial wallet team** if they are using `instant-seal-node` for tests.